db/migrate/20190923181352_create_external_registry_bikes.rb

```diff
@@ -0,0 +1,32 @@
+class CreateExternalRegistryBikes < ActiveRecord::Migration
+  def change
+    create_table :external_registry_bikes do |t|
+      t.string :external_registry_name, null: false, index: true
+      t.string :external_registry_url, null: false, index: true
+      t.integer :external_registry_country_id, null: false, index: true
```

Comment:
Not all external registries are country specific.

Reply:
Renamed to `country` in 48ece8a1
Should we anticipate bike records without a country? 🤔

Reply:
Probably not? At least not for stolen records